### ISSUE TITLE
Metallurgy: Classical Weaponry

### ISSUE THREAD

**Issue by Alziibun** [Alziibun/Perdition#17]
### Arming Sword
A widely known design used by knights and shown in fairy tales.  It predates the smallsword but looks nice!
- Single-handed
- A moderately balanced weapon, allowing for swift, decisive strikes or brutal thrusts
### Smallsword
Similar to a Rapier, which was used for duels of honor, it is instead a double-edged thin blade.
- Single-handed
- Light, not very strong slash but great stabbing
### Sabre
A widely known heavy sword used in cavalry battles.  Typically used to decapitate the enemy officer and in mount & blade tactics.
- Single-handed
- Quite heavy, it's not easy to swing
- Designed purely for decapitation and laceration

### Longsword
A solid and versatile choice for a two-handed long blade.  It can be used swiftly like a short sword in half-swording (requires armored hands) or its standard use in the longsword style (used when unarmored)
- Two-handed, reduced effectiveness single-handed
- Heavy, cannot be used with exhaustion unless half-swording
- The most ideal sword-type classical weapon for a zombie apocalypse given the proper setup.
- the playstyle would not be very different from spears in Zomboid

I'll add more later I'm sleepy

**Comment by pieetrr** [comment]
Would be very interesting if being able to smith these was the Metalworker's unique occupation bonus, or rejecting that, a blacksmith trait that you would have to pick to be able to make these.

> Two-handed, reduced effectiveness single-handed
> Heavy, cannot be used with exhaustion unless half-sword
> The most ideal sword-type classical weapon for a zombie apocalypse given the proper setup.
the playstyle would not be very different from spears in Zomboid

Longswords are not that heavy, on average around1.5 kg. Definitely nothing like a spear. Do you mean a greatsword by any chance?

**Comment by Alziibun** [comment]
> Would be very interesting if being able to smith these was the Metalworker's unique occupation bonus, or rejecting that, a blacksmith trait that you would have to pick to be able to make these.

Yes the plan is for Metallurgy to be a metalworker thing

> Longswords are not that heavy, on average around1.5 kg. Definitely nothing like a spear. Do you mean a greatsword by any chance?

Because of torque and leverage it is necessary to use techniques like halfswording.  It's in the same vein as katana where it is less effective single handled because you need 2 arms to balance the limb torque

**Comment by pieetrr** [comment]
> Because of torque and leverage it is necessary to use techniques like halfswording. It's in the same vein as katana where it is less effective single handled because you need 2 arms to balance the limb torque

Halfswording was used to pinpoint drive the blade between armour plates when you're wrestling with an armoured opponent and all you have is your sword. Most knights were smart enough to carry daggers on their belts instead but half swording works too. 
Unless you mean the half sword mode with the longsword would be used as a short spear gameplay wise i dont see how the normal longsword acts like a spear, nor why it should do so in a zombie setting where armour is not to be found on any zombie. Or you are just trying to make variety of gameplay styles with the same weapon, making it a bit unrealistic so it can server a dual purpose?

Also half swording doesnt require armoured hands. 

Forgot to mention that the arming sword and sabre are long blade weapons when in pz the machete is long blade, unless you want to be gamey which is fine by me. We can maybe move the machete to short blade.

**Comment by Alziibun** [comment]
>Halfswording was used to pinpoint drive the blade between armour plates when you're wrestling with an armoured opponent and all you have is your sword. Most knights were smart enough to carry daggers on their belts instead but half swording works too.
Yes, it was also used for pommel strikes.  On the other hand it allowed for a more quick and versatile fighting style.

>Unless you mean the half sword mode with the longsword would be used as a short spear gameplay wise i dont see how the normal longsword acts like a spear, nor why it should do so in a zombie setting where armour is not to be found on any zombie. Or you are just trying to make variety of gameplay styles with the same weapon, making it a bit unrealistic so it can server a dual purpose?
Spears have an animation where they do a quick swipe, I feel like it's close enough that we can just use the spear fighting code.

> Also half swording doesnt require armoured hands.
It does not, but is not recommended, as you can injure yourself in unexpected situations.  Without gloves the player will have a 10% chance at scratching themselves on a hit.

> Forgot to mention that the arming sword and sabre are long blade weapons when in pz the machete is long blade, unless you want to be gamey which is fine by me. We can maybe move the machete to short blade.
I'm not really sure why they consider machete a long blade, the short blade skill is vastly unused unless they want it to be a knife thing, which a smallsword would be just a longer dagger
